force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -350,19 +427,25 @@ public with sharing class FlowControlLoopsCollections {
         Map<String, Integer> employeeSalaries = new Map<String, Integer>();
 
         // Add key-value pairs to the Map
-        // add John Doe => 50000
-        // add Jane Smith => 60000
-        // add Sam Brown => 55000
-        // add Alice Johnson => 65000
+        employeeSalaries.put(John Doe, 50000);
+        employeeSalaries.put(Jane Smith, 60000);
+        employeeSalaries.put(Sam Brown, 55000);
+        employeeSalaries.put(Alice Johnson, 65000);
 
         // Initialize a list to store the names of high paid employees
-
+        List<String> listName = HighEarners<Stings>();
         // Iterate over the Map using a for loop
+        for (String employeeName : employeeSalaries.keySet()) {
+
             // Check if the salary of the employee is more than 55000
+         if (employeeSalaries.get(employeeName) > 55000) {
                 // Add the employee to the list of high paid employees
+                HighEarners.add(employeeName);
```

Comment:
```suggestion
                highEarners.add(employeeName);
```

Reply:
Stick to camelCase for your variable names. Since Apex is a strongly-typed language, it doesn't affect compilation, but it affects style and readability of code